### PR TITLE
OpenTelemetry 2020 Annual Review

### PR DESCRIPTION
**Pull request by ccaraman**
OpenTelemetry 2020 Sandbox Annual Review

### CODE REVIEW

File: reviews/2020-opentelemetry-annual.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+- C/C++ SDK
+- GoLang SDK
+- Java (SDK and Auto-Instrumentation)
+- JavaScript SDK
+- Python SDK
+- Ruby SDK
+- Erland/Elixir SDK
+- PHP SDK
+- Rust SDK
+- Swift SDK
```
Comment (alena1108) [inline reviews/2020-opentelemetry-annual.md:17]: Could you insert a link to the original Sandbox presentation? Similar to how it was done here https://github.com/cncf/toc/pull/440/files#diff-e9dc5bcb93599ba622b4e3e626fdd4eeR24

File: reviews/2020-opentelemetry-annual.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+## Comparison against last year
+*How has the project performed against its goals since the last review? (We won't penalize you if your goals changed for good reasons.)*
+
+OpenTelemetry joined the CNCF as a sandbox project in May of last year. There is no review to compare against. Please refer to highlights section for notable accomplishments since joining CNCF. 
+
+## Goals
+*What are the current goals of the project? For example, are you working on major new features? Or are you concentrating on adoption or documentation?*
+
+The following are “must-do” goals:
```
Comment (alena1108) [inline reviews/2020-opentelemetry-annual.md:84]: Can we also include the link to the Project Roadmap? OpenTelemetry GA Launch Plan
provides a great level of details, but I am not sure if it's the only doc communicating features development to the end users/contributors.

File: reviews/2020-opentelemetry-annual.md (hunk 1 above)
Comment (SergeyKanzhelev) [reply reviews/2020-opentelemetry-annual.md:17]: @ccaraman links:

- Slides: https://docs.google.com/presentation/d/1roUFtkiVw25N7nRuUnMa-ziNhb6vzhZAe43jpBfiDiw/edit#slide=id.g5724df7a45_0_258
- Proposal: https://github.com/cncf/toc/blob/f4ecf1bf391867d0840434b638cd9ef6cb2f57fe/proposals/sandbox/opentelemetry.adoc
- PR: https://github.com/cncf/toc/pull/233

**Comment by alena1108** [comment]
LGTM for Sandbox annual review.

File: reviews/2020-opentelemetry-annual.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+
+
+## Highlights from the past year
+
+OpenTelemetry declared Beta for the collector and five languages(Java, JavaScript, Python, Go, and .Net APIs and SDKs). The Beta milestone had several requirements such as
+Implementation of metrics, traces, context propagation and resource metadata for version <= 0.3 in the [spec](add link to repository)
+Documentation for getting started
+Ability to export in Jaeger, Prometheus and Zipkin
+Testing coverage and automate release process
+For more information on the Beta requirements and the process, please refer to the [Beta Launch Plan](https://docs.google.com/document/d/1eviggoIguOS89dgKL-8ntUOCfPP6FLWAoQkGoBDsld0/edit?usp=sharing)
```
Comment (yurishkuro) [inline reviews/2020-opentelemetry-annual.md:36]:
```suggestion
OpenTelemetry declared Beta for the collector and five languages(Java, JavaScript, Python, Go, and .Net APIs and SDKs). The Beta milestone had several requirements such as:
  * Implementation of metrics, traces, context propagation and resource metadata for version <= 0.3 in the [spec](add link to repository)
  * Documentation for getting started
  * Ability to export in Jaeger, Prometheus and Zipkin
  * Testing coverage and automated release process
  
For more information on the Beta requirements and the process, please refer to the [Beta Launch Plan](https://docs.google.com/document/d/1eviggoIguOS89dgKL-8ntUOCfPP6FLWAoQkGoBDsld0/edit?usp=sharing)
```

Comment (yurishkuro) [reply reviews/2020-opentelemetry-annual.md:36]: Also, what about `for version <= 0.3 in the [spec](add link to repository)` ?

File: reviews/2020-opentelemetry-annual.md [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+There are 9 project-wide maintainers on the governance committee
+- Ben Sigelman (Lightstep)
+- Bogdan Drutu (Splunk)
+- Constance Caramanolis (Splunk)
+- Liz Fong-Jones (Honeycomb)
+- Morgan James McLean (Google)
+- Sarah Novotny (Microsoft)
+- Sergey Kanzhelev (Google)
+- Ted Young(Lightstep)
+- Yuri Shkuro (Uber)
```
Comment (SergeyKanzhelev) [inline reviews/2020-opentelemetry-annual.md:68]: affiliation changed here

```suggestion
- Yuri Shkuro (Facebook)
```

Comment (yurishkuro) [inline reviews/2020-opentelemetry-annual.md:68]:
```suggestion
- Yuri Shkuro (Facebook)
```

**Pull request merged**